### ISSUE TITLE
【书籍】Grokking Deep Learning

### ISSUE THREAD

**Issue by howie6879** [howie6879/pylab#12]
阅读进度：

- [ ] 1.Introducing deep learning: why you should learn it?
- [ ] 2.Fundamental concepts: how do machines learn?
- [ ] 3.Introduction to neural prediction: forward propagation
- [ ] 4.Introduction to neural learning: gradient descent
- [ ] 5.Learning multiple weights at a time
- [ ] 6.Building your first deep neural network: introduction to backpropagation
- [ ] 7.How to picture neural networks: in your head and on paper
- [ ] 8.Learning signal and ignoring noise: introduction to regularization and batching
- [ ] 9.Modeling probabilities and nonlinearities: activation functions
- [ ] 10.Neural learning about edges and corners: intro to convolutional neural networks
- [ ] 11.Neural networks that understand language: king – man + woman == ?
- [ ] 12.Neural networks that write like Shakespeare: recurrent layers for variable-length data
- [ ] 13.Introducing automatic optimization: let’s build a deep learning framework
- [ ] 14.Learning to write like Shakespeare: long short-term memory
- [ ] 15.Deep learning on unseen data: introducing federated learning
- [ ] 16.Where to go from here: a brief guide
